### DISCUSSION TITLE
Probelmas conVue

### DISCUSSION THREAD

**Discussion by MarceloCortesC** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#462] [Q&A]
Por favor, alguien me puede ayudar a solucionar este error... no se , me aparece esto y en la pagina del navegador me aparece que el localhost rechazo la conexion, lo hice tal y como el mc lo ha
![2s](https://user-images.githubusercontent.com/99509464/159111517-020bb93d-2c8b-4912-9d33-44cdebdca5b8.JPG)
ce en la clase

**Comment by hackit92** [comment]
Creo que necesitaríamos más información, pero parece ser un error referente a la conexión

**Comment by doguedogue** [comment]
Checate esta solución de stackoverflow, le ha pasado a otros:

I also had this issue and none of the solutions listed were helpful. However the problem was rather easy to solve.
Just go into the Network tab of the Chrome Developer Console. Be sure that the connection is on No throttling and not Offline.

https://stackoverflow.com/questions/67191286/crbug-1173575-non-js-module-files-deprecated-chromewebdata-index%EA%9E%89530595551
